docker/run.yml

```
@@ -0,0 +1,17 @@
+# Referenced when running locally
+
+services:
+  backend:
+    networks:
+      # Needed to access MongoDB and Redis
+      shared_network:
+  minion:
+    networks:
+      # Needed to access MongoDB and Redis
+      shared_network:
+
+networks:
+  # This network allows access to shared services like MongoDB and Redis
+  shared_network:
+    name: shared_network
```

Shan't we add a project name or something like that ?

Otherwise you can't work on two clones of the repo at the same time.

I think the COMPOSE_PROJECT_NAME is shared with dependencies, right ?

In this case, prefix it:
```suggestion
    name: ${COMPOSE_PROJECT_NAME}_shared_network
```

Note: do it only if it's simple enough.